<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>journal_page</class>
 <widget class="QWidget" name="journal_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>470</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Journal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Reference File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>10</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>59</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Article Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="article_title_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>57</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>212</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Journal Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="journal_title_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Issue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="issue_number_line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="volume_number_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>283</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Year Published</string>
   </property>
  </widget>
  <widget class="QComboBox" name="reference_style">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_page_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_page_line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="author_checkBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>multiple-author</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contributor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="English_checkBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>494</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>74</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_citation_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Get Citation</string>
   </property>
  </widget>
  <widget class="QLabel" name="citationInPaperCLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>113</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>In-text Citation</string>
   </property>
   <property name="buddy">
    <cstring>inpaper_citation_line</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpaper_citation_line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>433</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bibliographyBLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>91</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="editor_label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </widget>
  <widget class="QLabel" name="translator_label">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translator</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="translator_multi">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mutiple-translator</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="editor_check">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_last">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translator_last_bar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="author_first">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_first_bar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="translator_last">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="editor_multi">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mutiple-editor</string>
   </property>
  </widget>
  <widget class="QLabel" name="editor_last">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editor_first_bar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="editor_first">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="translator_check">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_last_bar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="translator_first_bar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>169</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editor_last_bar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="translator_first">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>281</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>313</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Advanced Info</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="bibliographyBLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>381</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>article_title_line</tabstop>
  <tabstop>author_first_bar</tabstop>
  <tabstop>author_last_bar</tabstop>
  <tabstop>editor_first_bar</tabstop>
  <tabstop>editor_last_bar</tabstop>
  <tabstop>translator_first_bar</tabstop>
  <tabstop>translator_last_bar</tabstop>
  <tabstop>journal_title_line</tabstop>
  <tabstop>volume_number_line</tabstop>
  <tabstop>issue_number_line</tabstop>
  <tabstop>date_line</tabstop>
  <tabstop>start_page_line</tabstop>
  <tabstop>end_page_line</tabstop>
  <tabstop>bibliographyBLineEdit</tabstop>
  <tabstop>inpaper_citation_line</tabstop>
  <tabstop>get_citation_button</tabstop>
  <tabstop>reference_style</tabstop>
  <tabstop>back_button</tabstop>
  <tabstop>editor_multi</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>translator_check</tabstop>
  <tabstop>load_button</tabstop>
  <tabstop>editor_check</tabstop>
  <tabstop>author_checkBox</tabstop>
  <tabstop>English_checkBox</tabstop>
  <tabstop>translator_multi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
